packages/web/src/components/spa/JobsSection/jobs_section.css

```diff
@@ -1,5 +1,436 @@
+    }
+
+    .job-box {
+        height: 50vh;
+        width: 26vw;
+    }
+
+    button {
+        width: 100vw;
+        height: 7vh;
+        font-size: 1.4em !important;
+    }
+
+    .button-container span {
+        height: 4.5vh;
+        width: 7vw;
+        font-size: 0.8em !important;
+        font-weight: 500;
+    }
+
+    .photo img {
+        width: 24vw;
+        height: 12vh;
+    }
+}
+
+@media only screen and (min-width: 1000px) and (max-width: 1035px) {
+    .jobs-container {
+        height: 100vh;
+        width: 101vw;
+    }
+
+    .job-box {
+        width: 26vw;
+        height: 40vh;
+    }
+
+    .text,
+    .text-2 {
+        font-size: 0.8em;
+        text-align: justify;
+    }
+
+    .button-container span {
+        font-size: 0.7em !important;
+        height: 3.5vh;
+    }
+
+    button {
+        font-size: 1.3em !important;
+    }
+}
+
+@media only screen and (min-width: 600px) and (max-width: 999px) {
+    .jobs-container {
+        height: 405vh;
+        width: 101vw;
+    }
+
+    .jobs {
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .job-box {
+        width: 47vw;
+        height: 55vh;
+        margin-top: 10%;
+    }
+
+    .photo img {
+        width: 35vw;
+        height: 14vh;
+    }
+
+    .text,
+    .text-2 {
+        font-size: 1.2em;
+        text-align: justify;
+    }
+
+    .button-container span {
+        font-size: 1.2em !important;
+        height: 5.5vh;
+        width: 13vw;
+    }
+
+    button {
+        width: 100vw;
+        height: 7vh;
+        font-size: 1.4em !important;
+    }
+}
+
+@media only screen and (min-width: 600px) and (max-width: 783px) {
+    .jobs-container {
+        height: 395vh;
+    }
+
+    .job-box {
+        width: 58vw;
+        height: 56vh;
+    }
+
+    .photo img {
+        width: 45vw;
+        height: 14vh;
+    }
+
+    .button-container span {
+        font-size: 1.1em !important;
+        height: 5vh;
+        width: 14vw;
+    }
+}
+
+@media only screen and (min-width: 600px) and (max-width: 671px) {
+    .jobs-container {
+        height: 405vh;
+    }
+
+    .job-box {
+        height: 58vh;
+    }
+
+    .button-container span {
+        font-size: 1em !important;
+    }
+
+    .photo img {
+        width: 55vw;
+        height: 16vh;
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    .jobs-container {
+        height: 408vh;
+        width: 100vw;
+    }
+
+    .jobs {
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .job-box {
+        margin-top: 12%;
+        height: 58vh;
+        width: 85vw;
+    }
+
+    .photo img {
+        width: 77vw;
+        height: 18vh;
+    }
+
+    button {
+        width: 200vw;
+        height: 7vh;
+        font-size: 1.5em !important;
+    }
+
+    .button-container span {
+        height: 5vh;
+        width: 23vw;
+        font-size: 1.4em !important;
+    }
+}
+
+@media only screen and (min-width: 400px) and (max-width: 500px) {
```

Comment:
You don't need the `min-width` here. You have a query for `max-width: 400px` below which would take precedence.